prod.tfvars

```diff
@@ -1,2 +1,2 @@
-allowed_jurisdictions = "'ia', 'IA','civil', 'CIVIL','privatelaw', 'PRIVATELAW','publiclaw', 'PUBLICLAW','employment','EMPLOYMENT'"
+allowed_jurisdictions = "'ia', 'IA','civil', 'CIVIL','privatelaw', 'PRIVATELAW','publiclaw', 'PUBLICLAW','employment','EMPLOYMENT','sscs','SSCS'"
```

Comment:
This one will start listening to their Case Events and processing them.  So if they have already deployed a CCD Definition which publishes their events then this would make them go live immediately.  Do you know if they have done this already or not?

Reply:
I think they are planning to do it on the release day Ravi. I will confirm with their BA.